examples/profile-aws/controls/iam_users_console_users_mfa.rb

```diff
@@ -1,8 +1,8 @@
-control 'iam_users_console_users_mfa' do
-  title 'Users with console access must have MFA enabled'
+control "iam_users_console_users_mfa" do
+  title "Users with console access must have MFA enabled"
   impact 1.0
 
-  describe aws_iam_users.where { has_console_password and not has_mfa_enabled } do
-    its('entries') { should be_empty }
+  describe aws_iam_users.where { has_console_password && (not has_mfa_enabled) } do
```

Comment:
I'm curious if this works as expected. I know that `and` is used in lots of profiles and wonder if this is part of the rspec/inspec dsl and how this change affects that?